examples/visualize_features.py

```diff
@@ -198,7 +236,25 @@ def main(rank: int = 0, world_size: int = 1):
                 orig = cv2.cvtColor(images[i].permute(1, 2, 0).cpu().numpy(), cv2.COLOR_RGB2BGR)
 
                 cv2.imwrite(f'{dirs["orig"]}/vis_{ctr}.jpg', orig * 255)
-                cv2.imwrite(f'{dirs["viz"]}/vis_{ctr}.jpg', colored[-1] * 255)
+
+                if args.intermediates:
```

Comment:
While you're in the code, can you add `interpolation='bilinear'` to `get_pca_map` a couple lines above this?

Reply:
Done!